### PR TITLE
build-review-rework

### PR DESCRIPTION
**Pull request by sedroche**

### CODE REVIEW

File: app/views/mobile-client-builds-row.html [hunk 1]
```diff
@@ -41,6 +41,9 @@
               kind="{{row.apiObject.kind}}"
               stay-on-current-page="true"></delete-link>
           </li>
+          <li ng-if="row.buildConfigsVersion | canI : 'update'">
+            <a ng-href="{{row.apiObject | editYamlURL}}" role="button">Edit YAML</a>
```
Comment (JameelB) [inline app/views/mobile-client-builds-row.html]: There's an `edit` action which brings you to a form which edits the pipeline. Is it possible to bring that in here as well? 

Can we move this above the delete action just to be consistent with the action list in the pipelines itself. 
image

File: app/views/mobile-client-builds-row.html [hunk 2]
```diff
@@ -50,30 +53,30 @@
...
         <div class="row">
-          <div class="col-md-2">
-            <b>Jenkins Job Name</b>
+          <div class="col-sm-6 col-md-2">
+            <b>Build Platform</b>
           </div>
-          <div class="col-md-4">
-            {{row.apiObject.metadata.name}}
+          <div class="col-sm-6 col-md-4">
+            {{row.apiObject.spec.buildPlatform}}
```
Comment (JameelB) [inline app/views/mobile-client-builds-row.html]: I think buildPlatform in spec is not supported by the API. We may need to add this as a label instead.

File: app/styles/_mobile-clients.less [hunk 3]
```diff
@@ -12,15 +12,45 @@
...
+
+          .state {
+            display: flex;
+            flex-wrap: wrap;
+          }
+
+          .download-trigger {
+            align-items: center;
+            display: flex;
+          }
```
Comment (JameelB) [inline app/styles/_mobile-clients.less:53]: Need to update css to be in alphabetical order

File: app/views/browse/mobile-clients.html [hunk 4]
```diff
@@ -76,9 +76,31 @@ <h3>Mobile Client Config</h3>
...
             </uib-tab>
-            <uib-tab ng-if="ctrl.mobileCIEnabled && ctrl.mobileClient.spec.clientType !== 'xamarin'" active="selectedTab.builds">
+            <uib-tab active="selectedTab.builds">
               <uib-tab-heading>Builds</uib-tab-heading>
-                <mobile-client-builds-list mobile-client="ctrl.mobileClient"></mobile-client-builds-list>
+                <div class="note" ng-if="(ctrl.mobileCIProvisioning || ctrl.mobileCIDeprovisioning) && ctrl.mobileClient.spec.clientType !== 'xamarin'">
+                  <div class="col-md-offset-1 col-md-10">
+                    <div class="alert alert-info">
+                      <span class="pficon pficon-info"></span>
+                      <span ng-class="{'spinner spinner-xs spinner-inline': (ctrl.mobileCIProvisioning || ctrl.mobileCIDeprovisioning) && ctrl.mobileClient.spec.clientType !== 'xamarin'}" aria-hidden="true"></span>
```
Comment (JameelB) [inline app/views/browse/mobile-clients.html]: Xamarin check not needed here as it is already checked in outer div

File: app/views/mobile-client-builds-row.html (hunk 1 above)
Comment (sedroche) [reply app/views/mobile-client-builds-row.html]: Ill leave this out for now as we may need to implement our own edit page

**Pull request merged**